types/uid-generator/tsconfig.json

```diff
@@ -4,6 +4,7 @@
         "lib": [
             "es6"
         ],
+        "target": "es6",
```

Comment:
Maybe the tests would pass if we don't specify the `target` parameter at all and let the test script choose x)

Reply:
Tried, unfortunately it doesn't work. I add the target option because of the conflict between CommonJS and ES6 module style. It seems only with this option syntax such as Export as Default could work properly.

Reply:
The bug (test script insists to install TypeScript 3.7 witch does not exist) could be fixed very soon. Because it may cause failures for all PRs with then entire @Types packages.

It most likely be caused by someone in TypeScript team who deleted 3.7 by accident.

Reply:
Someone already created a issue <https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40054>

Reply:
Same bug at: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/39973#issuecomment-548502541

Reply:
Alright, let's wait then